### ISSUE TITLE
Delete duplicate demos

### ISSUE THREAD

**Issue by rachel-fenichel** [google/blockly#6065]
The demos in this repository show how to use pieces of Blockly's APIs. They predate the blockly-samples repository. We moved the demos that don't use storage into the examples folder on blockly-samples and you can explore them on the [samples page](https://google.github.io/blockly-samples/).

I added redirects in https://github.com/google/blockly/pull/6045, so we can the duplicate demos.

Code and block factory both use storage and are not part of this bug.

**Comment by rachel-fenichel** [comment]
After #6067 there are a few demos that are not linked but still exist in this repo, and need decisions:

- Old block factory
- Minimap
- Mobile
